packages/@ember/routing/index.ts

```diff
@@ -1 +0,0 @@
-export { LinkComponent as LinkTo } from '@ember/-internals/glimmer';
```

Comment:
Per the RFC at https://emberjs.github.io/rfcs/0671-modernize-built-in-components-1.html, the `LinkTo` export here should be an opaque thing usable in strict mode. This should be updated to reflect that goal (it should re-export `LinkTo`).

The same for `Input` and `Textarea` should be audited.

Reply:
Done.